.pre-commit-config.yaml

```diff
@@ -19,7 +19,15 @@ repos:
     -   id: frontend-lint
         name: Linting of JS, Vue, SCSS and CSS files
         description: This hook handles all frontend linting for Kolibri
-        entry: yarn run lint -w
+        entry: yarn workspace kolibri-tools run lint --write
```

Comment:
This is not a valid command. The only way to invoke this directly is `node ./packages/kolibri-tools/lib/cli.js lint --write`.

I think we may have discovered why I had a base lint command in the package.json - because that could be written as `kolibri-tools lint`.

Reply:
This is the same pattern we're currently using for the `build-kolibri-tools` command: https://github.com/learningequality/kolibri/blob/develop/package.json#L31

Reply:
Yes, because `build` is a valid command inside the `kolibri-tools` package.json - note that this is for building the distributable version of `kolibri-tools` not for invoking the `kolibri-tools` `build` subcommand.

Reply:
ah, I see. Maybe we could call the bin alias `kolibri-tools-cli` to disambiguate?

> The only way to invoke this directly is `node ./packages/kolibri-tools/lib/cli.js lint --write`

I believe `yarn run kolibri-tools lint --write` should work too since `kolibri-tools` is getting registered as a runnable binary?

Reply:
No, I don't believe so.

`yarn/npm run ...` is only used to invoke registered scripts in a package.json, whereas if `kolibri-tools` is registered as a runnable libary it is invoked by `kolibri-tools` (although command line invocation mileage varies - but it works consistently when invoked inside a package.json command).